src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java

```diff
@@ -101,6 +101,8 @@ public class IWantDocument extends FinancialSystemTransactionalDocumentBase impl
     private String processorNetId;
     private String processorName;
 
+    private String contractIndicator;
```

Comment:
Good question. Initially Nancy implemented this as a boolean but we had trouble getting this to work with the radio button. We found examples in the code where this was defined as a String and switching to a String worked in this case too. I could create a separate ticket to further look into changing this to a boolean if you think that would be better.

Reply:
Thanks for the explanation. I'm fine with leaving it as a string for now, but I would suggest creating a follow-up user story to look into redefining that property and the similar property as booleans. The future user story may need be set them up as Boolean wrapper objects instead of primitive booleans.